devops/ansible/roles/large_image/tasks/svs.yml

```diff
@@ -0,0 +1,28 @@
+---
+- name: Install svs tile source system dependencies
```

Comment:
I think this entire file can turn into just installing the apt package `libopenslide-dev`, then installing the pip package `openslide-python`. I believe all of these other apt packages were for building OpenSlide from source, and Pillow will now always be installed for `large_image`.